arch/arm/src/kinetis/kinetis_edma.h

```diff
@@ -0,0 +1,455 @@
+/****************************************************************************
+ * arch/arm/src/kinetis/kinetis_edma.h
+ *
+ *   Copyright (C) 2019 Gregory Nutt. All rights reserved.
+ *   Author: Gregory Nutt <gnutt@nuttx.org>
+ *
+ * This file was leveraged from the NuttX i.MXRT port.
+ * Portions of that eDMA logic derived from NXP sample code which has
+ * a compatible BSD 3-clause license:
+ *
```

Comment:
Please include the Apache License here and keep the Freescale/NXP BSD License below. This is how Samsung did when they released TIzenRT as Apache using NuttX BSD License.

Reply:
@acassis do we have a CCLA from NXP? I thought this was still tied up and should not be changed until it was resolved.